homeassistant/components/weheat/sensor.py

```diff
@@ -206,20 +206,71 @@ class WeHeatSensorEntityDescription(SensorEntityDescription):
 
 ENERGY_SENSORS = [
     WeHeatSensorEntityDescription(
-        translation_key="electricity_used",
-        key="electricity_used",
+        translation_key="electricity_used_heating",
+        key="electricity_used_heating",
         native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
         device_class=SensorDeviceClass.ENERGY,
         state_class=SensorStateClass.TOTAL_INCREASING,
-        value_fn=lambda status: status.energy_total,
+        value_fn=lambda status: status.energy_in_heating,
     ),
     WeHeatSensorEntityDescription(
-        translation_key="energy_output",
-        key="energy_output",
+        translation_key="electricity_used_cooling",
+        key="electricity_used_cooling",
         native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
         device_class=SensorDeviceClass.ENERGY,
         state_class=SensorStateClass.TOTAL_INCREASING,
-        value_fn=lambda status: status.energy_output,
+        value_fn=lambda status: status.energy_in_cooling,
+    ),
```

Comment:
Okay so this PR now changes description keys. This means the unique id changes as that uses the description keys. This means that new entities are created, and we should not do that. So the options are
1. Keep using the old keys
2. Migrate them

I'd go for 1 as migrating is a PITA

Reply:
The old ones are the total for all types combined, the new ones are for specific types only. 
Not sure if migrating is required as the values are unavailable for the last year or so… but might be good to just keep them besides the new ones, especially if they start working again with the new SDK now.